### ISSUE TITLE
Adding role="switch"

### ISSUE THREAD

**Issue by obliviga** [minhur/bootstrap-toggle#120]
Should be adding the attribute `role="switch"` for accessibility purposes. Please see https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_switch_role

**Comment by pvagner** [comment]
I am afraid adding the aria role is not going to magically bring complete accessibility support. A proper keyboard, focus events and complementary styling would be needed. I have googled this issue because I have just experienced usage of this component contributes to noticeable accessibility related issues with [Known](https://withknown.com/) powered sites.
Is there a way to improve on this front? I am entirely new to the world of Bootstrap components but is a reference to [Bootstrap Accessibility Plugin](https://github.com/paypal/bootstrap-accessibility-plugin) relevant here?
